src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -208,9 +208,11 @@ struct Metavariables {
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
         tmpl::pair<DomainCreator<volume_dim>, domain_creators<volume_dim>>,
+        tmpl::pair<StepChooser<StepChooserUse::LtsStep>, tmpl::list<>>,
+        tmpl::pair<StepChooser<StepChooserUse::Slab>, tmpl::list<>>,
```

Comment:
The initialization list includes `Initialization::Actions::TimeAndTimeStep<Metavariables>`, which is the action that now puts the step choosers in the box. I think that action is needed because of the dependencies that the domain has on time when using time-dependent maps, but I'm not certain.

Reply:
OK.  Yes, the action is needed for the reason you say.